distributed-lock-core/src/main/java/com/github/alturkovic/lock/advice/LockMethodInterceptor.java

```diff
@@ -59,6 +59,12 @@ public Object invoke(final MethodInvocation invocation) throws Throwable {
     final LockContext context = new LockContext(invocation);
     try {
       return executeLockedMethod(invocation, context);
+    } catch (DistributedLockException e) {
+      if (context.getLocked() != null && !context.getLocked().throwing()) {
+        log.warn("Cannot obtain lock for keys {} in store {}", context.getKeys(), context.getLocked().storeId(), e);
+        return null;
```

Comment:
Please add this part to the docs and the annotation explaining that `null` will be returned from non-void methods in case of a failed lock.